frameworks/PHP/reactphp/reactphp.dockerfile

```diff
@@ -1,29 +1,33 @@
-FROM ubuntu:22.04
+FROM ubuntu:24.04
 
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update -yqq && apt-get install -yqq software-properties-common > /dev/null
-RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php
+RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php > /dev/null
 RUN apt-get update -yqq > /dev/null && \
     apt-get install -yqq git unzip wget curl build-essential \
-    php8.3-cli php8.3-mbstring php8.3-dev php8.3-xml > /dev/null
+    php8.4-cli php8.4-mbstring php8.4-dev php8.4-xml > /dev/null
 
 # An extension is required!
 # We deal with concurrencies over 1k, which stream_select doesn't support.
-RUN apt-get install -yqq libuv1-dev > /dev/null \
-    && pecl install uv-beta > /dev/null \
-    && echo "extension=uv.so" > /etc/php/8.3/cli/conf.d/uv.ini
+# libuv
+# RUN apt-get install -yqq libuv1-dev > /dev/null \
+#     && pecl install uv-beta > /dev/null \
+#     && echo "extension=uv.so" > /etc/php/8.4/cli/conf.d/uv.ini
+
+# libevent
+RUN apt-get install -y libevent-dev > /dev/null \
+    && pecl install event-3.1.4 > /dev/null \
+    && echo "extension=event.so" > /etc/php/8.4/cli/conf.d/event.ini
```

Comment:
Purely out of interest, why the switch to `event` from `uv`?

Reply:
Oh sorry !

I test it locally also with libevent, to compare with other frameworks that use it too.
Libuv is only used for Reactphp, not other php frameworks, so we can compare with others.

Also it's good to compare with both event systems and check the faster one.

Reply:
With libevent other PHP frameworks have up to 3,854,134 req/s with the plaintext test.
And Reactphp with libuv have a max of ~70,000 req/s.

So we can check if it's a problem with the event system.

Reply:
> Oh sorry !

Please, no need to be sorry!

> With libevent other PHP frameworks have up to 3,854,134 req/s with the plaintext test. And Reactphp with libuv have a max of ~70,000 req/s.
> 
> So we can check if it's a problem with the event system.

That be pretty neat! Also, the current set up is fully single process, will have a look at some point at how easy it is to rework it into multi core support.